src/apps/routes/admin.clj

```diff
@@ -129,7 +138,7 @@
 
   (context "/:system-id/:app-id" []
     :path-params [system-id :- SystemId
-                  app-id    :- AppIdJobViewPathParam]
+                  app-id    :- StringAppIdParam]
```

Comment:
These were the same schema, so I just deleted `AppIdJobViewPathParam` and used `StringAppIdParam` that was already in the common swagger lib.